src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/RazorEventSource.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.AspNetCore.Razor;
+
+[EventSource(Name = "RazorEventSource")]
```

Comment:
The alternative to this would just be to steal Roslyn's event source name, or even just use the code markers guid. Not sure what the etiquette there is :)

Reply:
Creating a new `EventSource` makes sense, it makes it easier not to accidently collide with their events.